ml-agents/mlagents/trainers/ppo/trainer.py

```diff
@@ -86,18 +86,30 @@ def _process_trajectory(self, trajectory: Trajectory) -> None:
 
         for name, v in value_estimates.items():
             agent_buffer_trajectory[f"{name}_value_estimates"].extend(v)
-            self._stats_reporter.add_stat(
-                self.optimizer.reward_signals[name].value_name, np.mean(v)
-            )
+            if hasattr(self.optimizer.reward_signals[name], "value_name"):
+                self._stats_reporter.add_stat(
+                    self.optimizer.reward_signals[name].value_name, np.mean(v)
```

Comment:
Will not all reward signals have `value_name`? Should we just remove it completely and just use the `name`?

Reply:
That would work too